jbmc/regression/jbmc/deterministic_assignments_json/Empty.java

```diff
@@ -0,0 +1,3 @@
+public class Empty {
```

Comment:
Rather that emptiness, is having or not having static fields the critical attribute?

Reply:
Yes, I've clarified that in the class names